### ISSUE TITLE
Add results to sotabench

### ISSUE THREAD

**Issue by Randl** [ultralytics/yolov5#39]
## 🚀 Feature
Add the model results on COCO to sotabench

## Motivation

[Sotabench](https://sotabench.com/benchmarks/object-detection-on-coco-minival) is arguably the best independent model benchmark which allows to obtain fair comparison between models. 

## Pitch

What is required is to add `sotabench.py` to the repository. The website has both guides and code examples. Existing submissions can be used as a guideline too.

**Comment by glenn-jocher** [comment]
Hello, thank you for your interest in our work! We have noticed that this issue is very similar to one or more existing issues, and thus we are closing this as a duplicate issue. For help please see the following issue(s):

- https://github.com/ultralytics/yolov5/issues/38

**Comment by glenn-jocher** [comment]
@Randl I've created a sotabench.py file now in the root directory, which essentially a slight modification of the existing test.py file. sotabench.py defaults to testing YOLOv5s. I've registered the repo on sotabench.py, but something is not working correctly, as all builds get skipped. Any ideas why this might be happening?

<img width="1194" alt="Screen Shot 2020-09-03 at 1 26 41 AM" src="https://user-images.githubusercontent.com/26833433/92091692-f0867b80-ed85-11ea-86e5-837b98a14ef5.png">
